docs/r2/hyperparameter_tuning_with_hparams.ipynb

```diff
@@ -0,0 +1,599 @@
+{
```

Comment:
<p>nit: for clarity, I suggest separating these into:</p><p><br></p><p>default python imports</p><p><br></p><p>tensorflow imports</p><p><br></p><p>other imports.</p><p><br></p><p>So:</p><p><br></p><p>import datetime</p><p><br></p><p>import tensorflow as tf</p><p><strong>from</strong> <strong>tensorboard.plugins.hparams</strong> <strong>import</strong> api_pb2</p><p><strong>from</strong> <strong>tensorboard.plugins.hparams</strong> <strong>import</strong> summary <strong>as</strong> hparams_summary</p><p><br></p><p><strong>from</strong> <strong>google.protobuf</strong> <strong>import</strong> struct_pb2</p><p><br></p><p><br></p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Updated.